src/Editor/GUI/Editors/EntityEditor.cpp

```diff
@@ -411,13 +411,48 @@ void EntityEditor::ScriptEditor(Component::Script* script) {
 
             for (int n = 0; n < propertyCount; n++) {
 
+                std::string propertyName = script->instance->GetPropertyName(n);
                 int typeId = script->instance->GetPropertyTypeId(n);
                 void *varPointer = script->instance->GetAddressOfProperty(n);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'varPointer' is assigned a value that is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreadVariable)